code/js/i18n/cs.json

```diff
@@ -0,0 +1,25 @@
+{
+  "@metadata": {
+    "author": "Roman",
+    "locale": "cs"
+  },
+  "rcalc-title": "Kalkulačka",
+  "rcalc-h1": "Romanova Kalkulačka",
+
+  "rcalc-dropdownButtonColor": "Barva",
+  "rcalc-dropdownContentDefaultTheme": "Víchozí téma",
```

Comment:
Mistake in Czech language.

Reply:
this is a brilliant example of me misspelling a czech word výchozí and not even knowing it is wrong, still I need to improve my czech haha